packages/orchestration/src/examples/shared/chain-hub-cf.js

```diff
@@ -0,0 +1,50 @@
+import { M } from '@endo/patterns';
+import { heapVowE } from '@agoric/vow/vat.js';
+import { CosmosChainInfoShape } from '../../typeGuards.js';
+
+/**
+ * @import {Zone} from '@agoric/zone';
+ * @import {CosmosChainInfo, IBCConnectionInfo} from '@agoric/orchestration';
+ * @import {ChainHub} from '../../exos/chain-hub.js';
+ */
+
+/**
+ * @param {Zone} zone
+ * @param {ChainHub} chainHub
+ */
+export const prepareChainHubCreatorFacet = (zone, chainHub) => {
+  const ConnectionInfoShape = M.record(); // TODO
+  const makeCreatorFacet = zone.exoClass(
```

Comment:
why an exo class instead of a simple `zone.exo`? There will be only one. I think you don't want a prepare/make and instead just export the simple make